### DISCUSSION TITLE
Caused by: io.debezium.DebeziumException: Online REDO LOG files or archive log files do not contain the offset scn 714273824.  Please perform a new snapshot.

### DISCUSSION THREAD

**Discussion by HQLboy** [apache/flink-cdc#713] [Q&A]
flinkcdc 同步oracle里的数据，一个job里select oracle里的9张表，insert intokafka里的9张表，为什么会报这个错呢，Caused by: io.debezium.DebeziumException: Online REDO LOG files or archive log files do not contain the offset scn 714273824.  Please perform a new snapshot.

**Comment by ghost** [comment]
你好, 请教一下问题怎么解决的记得吗?

**Reply by man-laughing** [reply]
我也遇到了这个问题，使用的是apache seatunnel 2.3.8，上游使用的是OracleCDC，下游是MYSQL，也是遇到这个错误job停止了；你后面有什么解决方案吗？
